cli/internal/upgrade/terraform.go

```diff
@@ -120,8 +120,7 @@ func (u *TerraformUpgrader) PlanTerraformMigrations(ctx context.Context, opts Te
 // aborted by the user.
 func (u *TerraformUpgrader) CleanUpTerraformMigrations(fileHandler file.Handler, upgradeID string) error {
 	cleanupFiles := []string{
-		filepath.Join(constants.UpgradeDir, upgradeID, constants.TerraformUpgradeBackupDir),
-		filepath.Join(constants.UpgradeDir, upgradeID, constants.TerraformUpgradeWorkingDir),
+		filepath.Join(constants.UpgradeDir, upgradeID),
```

Comment:
is it safe to not specify the subdirs anymore? not saying it is not, but just want to raise potential implications if there could be more directories. We still seem to follow explicit listing in l.97f

Reply:
Yeah, I should've changed this when we implemented the upgrade versioning. Before we had the 2 subfolders that needed to be cleaned individually. Now we have 1 folder `{constants.UpgradeDir}/{upgradeID}`